src/hooks/dark-mode.js

```diff
@@ -10,6 +10,7 @@ export default function useDarkMode() {
     root.classList.remove(colorTheme);
     root.classList.add(theme);
     localStorage.setItem('theme', theme);
+    window.dispatchEvent(new Event('themeChanged'))
```

Comment:
I know we already talked about this, but I've been thinking about it and I think we can use React's Context API instead of the dispatching this event. I don't think we need to do this right now since this works, but we should circle back to this in the near future.